src/ui.cpp

```diff
@@ -629,17 +627,18 @@ void UpdateDialog::OnClose(wxCloseEvent&)
 }
 
 
-void UpdateDialog::OnSkipVersion(wxCommandEvent&)
+void UpdateDialog::OnRemindOneHour(wxCommandEvent&)
 {
-    Settings::WriteConfigValue("SkipThisVersion", m_appcast.Version);
+    static const int ONE_HOUR_IN_SECONDS = 3600;
+    Settings::WriteConfigValue("UpdateInterval", ONE_HOUR_IN_SECONDS);
```

Comment:
The default is what we set in our application when we initialize Winsparkle (24 hours).
Beyond that, if the user  clicks on Remind in an hour, then the next clicks the Close button, the update interval will be 1 hour. I explained this behavior to Amber and she was OK with it.